### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by JCook21**
This PR updates the easy, point releases for out of date dependencies in the project. It also includes some small test fixes as a result of this.

### CODE REVIEW

File: tests/e2e/specs/BoundaryEventCommonBehaviour.spec.js [hunk 1]
```diff
@@ -98,7 +95,7 @@ boundaryEventData.forEach(({ type, nodeType, eventXMLSnippet, eventXMLSnippetWit
         .its('xml')
         .then(removeIndentationAndLinebreaks)
         .then(xml => {
-          const boundaryEventPositionXml = 'bpmndi:BPMNShape id="node_3_di" bpmnElement="node_3"><dc:Bounds x="662" y="481"';
+          const boundaryEventPositionXml = 'bpmndi:BPMNShape id="node_3_di" bpmnElement="node_3"><dc:Bounds x="662" y="471"';
```
Comment (JCook21) [inline tests/e2e/specs/BoundaryEventCommonBehaviour.spec.js]: Not sure why the 10px difference but changing this allowed the test to pass.

**Comment by JCook21** [comment]
Not totally sure why but I'm having some formatting issues in PHPStorm. I suspect it's related to having had two projects with different code styles open in the same window. I'll look into it and will correct the problems.

File: tests/e2e/specs/Modeler.spec.js [hunk 2]
```diff
@@ -168,22 +166,14 @@ describe('Modeler', () => {
     const poolPosition = { x: 150, y: 300 };
     dragFromSourceToDest(nodeTypes.pool, poolPosition);
 
-    getElementAtPosition(startEventPosition)
-      .then(getLinksConnectedToElement)
-      .then($links => $links[0])
-      .then(isElementCovered)
-      .should(isCovered => expect(isCovered).to.be.false);
+    getElementAtPosition(startEventPosition).then(getLinksConnectedToElement).then($links => $links[0])
```
Comment (lisa-fehr) [inline tests/e2e/specs/Modeler.spec.js]: It seems easy to read when on their own lines throughout the pr. Thoughts?

File: tests/e2e/specs/BoundaryEventCommonBehaviour.spec.js (hunk 1 above)
Comment (lisa-fehr) [reply tests/e2e/specs/BoundaryEventCommonBehaviour.spec.js]: #741 should hopefully fix this

Comment (JCook21) [reply tests/e2e/specs/BoundaryEventCommonBehaviour.spec.js]: @lisa-fehr awesome. For some reason this change passed on my machine but failed on CircleCI...

**Pull request merged**